nominees/datawinners.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "DataWinners",
+  "description": "DataWinners is a data collection application supporting multiple channels like sms, web and smartphone. Goal is to enable seamless data collection, reporting and visualization.",
+  "website": "http://datawinners.com/",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "http://www.apache.org/licenses/LICENSE-2.0"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 4,
+      "evidenceURL": "https://allchildrenreading.org/innovator/human-network-international/"
+    },
+    {
+      "SDGNumber": 9,
+      "evidenceURL": "https://www.slideshare.net/SandraGubler1/human-network-international-its-services"
+    },
+    {
+      "SDGNumber": 10,
+      "evidenceURL": "https://www.slideshare.net/SandraGubler1/human-network-international-its-services"
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/mangroveorg/datawinners",
+  "organizations": [
```

Comment:
@rachita19082 would it be possible to add a website field for this organization entry. Thanks.

Reply:
Resolved in commit 4e3d973